.github/workflows/pr.yaml

```diff
@@ -288,16 +269,28 @@ jobs:
       with:
         path: ~/.planemo
         key: planemo_cache_py_${{ matrix.python-version }}_gxy_${{ needs.setup.outputs.galaxy-head-sha }}
+    - name: Prepare specific job_conf for Sirius
+      uses: danielr1996/envsubst-action@1.0.0
+      env:
+        USERNAME_SIRIUS: "${{ secrets.USERNAME_SIRIUS }}"
+        PASSWORD_SIRIUS: "${{ secrets.PASSWORD_SIRIUS }}"
+      with:
+        input: "${{ github.workspace }}/config/job_conf_planemo.xml"
+        output: "${{ github.workspace }}/config/job_conf_planemo_subst.xml"
     - name: Planemo test
       uses: galaxyproject/planemo-ci-action@v1
       id: test
+      env:
+        USERNAME_SIRIUS: "${{ secrets.USERNAME_SIRIUS }}"
+        PASSWORD_SIRIUS: "${{ secrets.PASSWORD_SIRIUS }}"
       with:
         mode: test
         repository-list: ${{ needs.setup.outputs.repository-list }}
         galaxy-fork: ${{ env.GALAXY_FORK }}
         galaxy-branch: ${{ env.GALAXY_BRANCH }}
         chunk: ${{ matrix.chunk }}
         chunk-count: ${{ needs.setup.outputs.chunk-count }}
+        additional-planemo-options: --job_config_file "${{ github.workspace }}/config/job_conf_planemo_subst.xml"
```

Comment:
In case you want to debug `danielr1996/envsubst-action@1.0.0` you could just execute the command  used in the action: https://github.com/danielr1996/envsubst-action/blob/b10d6e6eb5dba1c22527571460ceb83bc17c0b28/entrypoint.sh#L3